src/function/algebra/simplify.js

```diff
@@ -447,6 +447,11 @@ export const createSimplify = /* #__PURE__ */ factory(name, dependencies, (
     }
 
     if (isAssociative(newRule.l, context)) {
+      const nonCommutative = !isCommutative(newRule.l, context)
+      let leftExpandsym
+      // Gen. the LHS placeholder used in this NC-context specific expansion rules
+      if (nonCommutative) leftExpandsym = _getExpandPlaceholderSymbol()
```

Comment:
@josdejong - I have left that in for now based on the explanation; but feel free to draw this logic into if (nonCommutative) {..} upon merging this request 👍

Reply:
👍  let's keep it here, it's fine and your explanation makes sense